java/ql/test/library-tests/dataflow/synth-global/test.ql

```diff
@@ -1,4 +1,5 @@
 import java
+import semmle.code.java.dataflow.ExternalFlow
```

Comment:
same

Reply:
This was originally exposed through `InlineFlowTest`, which it is no longer. It's needed later in this file for `ModelValidation`.